sdk/containerregistry/Microsoft.Azure.ContainerRegistry/CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Release History
+
+## 1.0.0-preview.1
+
+- Require `System.IdentityModel.Tokens.Jwt >= 5.1.2` instead of `System.IdentityModel.Tokens.Jwt == 5.1.2`.
+
```

Comment:
Consider adding some note here about "Updating package version to follow standard Azure-SDK-for-Net versioning guidelines"